hardware/computemodule/cm-emmc-flashing.md

```diff
@@ -118,6 +118,22 @@ The `/dev/sdX1` and `/dev/sdX2` partitions can now be mounted normally.
 
 Make sure J4 (USB SLAVE BOOT ENABLE) / J2 (nRPI_BOOT) is set to the disabled position and/or nothing is plugged into the USB slave port. Power cycling the IO board should now result in the Compute Module booting from eMMC.
 
+## Flashing the bootloader EEPROM - Compute Module 4
+The `rpiboot` tool is the recommended method for updating the bootloader EEPROM on Compute Module 4. After following the initial EMMC flashing setup steps run the following command to run the `recovery` image instead of the EMMC image.
```

Comment:
How does the recovery image know what EEPROM image to flash?
(I don't have a CM4, so apologies if my questions don't make sense)

Reply:
It flashes the one in the recovery folder